serverless/email/Makefile

```diff
@@ -0,0 +1,4 @@
+.PHONY: build
+
+build:
+	sam build
```

Comment:
Not sure where this should go, but we should add in information about deploying DynamoDB locally for local development (ref: https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/DynamoDBLocal.DownloadingAndRunning.html#docker). 

If we don't want to do that, then we should definitely mock the AWS service client operations - https://aws.github.io/aws-sdk-go-v2/docs/unit-testing/